index.html

```diff
@@ -251,6 +251,58 @@ <h3 class="semi-bold center-text work-title">Skills</h3>
           </div>
         </div>
       </section>
+      
+      <section id="contact" class="flex-column container center-align">
+        <h2 class="bold big-font center-text color1">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+
+        <form
+          action="#"
+          method="post"
+          id="contact-form"
+          class="flex-column center-align"
+        >
+          <input type="text" name="name" placeholder="Full name" required />
+          <input
+            type="email"
+            name="email"
+            placeholder="Email address"
+            required
+          />
+          <textarea name="comment" id="comment" cols="30" rows="10" required>
+Enter text here</textarea
+          >
+          <button type="submit" class="btn">Get in touch</button>
+        </form>
```

Comment:
- [ ] Kindly submit your form to [formspree service](https://formspree.io/html) as stated in the project requirements